tests/test_vcs_git/test_vcs_git.py

```diff
@@ -13,27 +9,40 @@ class GitTests(DxrInstanceTestCaseMakeFirst):
     def test_diff(self):
         """Make sure the diff link exists and goes to the right place."""
         response = self.client().get('/code/source/main.c')
-        ok_('/commit/cb339834998124cb8165aa35ed4635c51b6ac5c2" title="Diff" class="diff icon">Diff</a>' in response.data)
+        ok_('/commit/5e2b2b554eb86f90e189217fa9dc2eba66259910" title="Diff" class="diff icon">Diff</a>' in response.data)
```

Comment:
It might be nice to factor this out as a name: `LATEST_REV` or something. Then we wouldn't have to touch every test (like you had to) whenever we change anything. Plus, the semantics would be clear.